rust/rebase/src/statement/poap_ownership.rs

```diff
@@ -3,20 +3,23 @@ use crate::types::{
     enums::subject::Subjects,
     error::StatementError,
 };
+use chrono::DateTime;
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Deserialize, JsonSchema, Serialize)]
 #[serde(rename = "statement")]
 pub struct PoapOwnership {
-    pub event_id: u64,
+    pub event_id: i64,
```

Comment:
can POAP event ids be negative numbers?

Reply:
This actually has to do with a downstream issue with JSON schema validation, the Rust JSON schema library uses a marker to say the number must be greater than -1, but the JS JSON schema library can't read that marker. Switching it to i64 fixes is for the JS JSON schema lib, though that's not an ideal route. 

On the JS side, I'm using [ajv](https://ajv.js.org/) along with [ajv formats](https://ajv.js.org/packages/ajv-formats.html), but it still didn't work. I could try to dig deeper into getting all this to play nice, but for now I'm just using i64, because the only result of sending a negative event ID is getting a not found.

I will add comments though, there's no way to know that's why it's an i64 from looking at the code.

Reply:
fair enough, we don't have to spend time fixing the schema lib so this makes sense